rtb/core/openrtb.hpp

```diff
@@ -108,8 +108,8 @@ namespace openrtb {
     struct Banner {
         ~Banner() {}
 
-        int w{};                     ///< Width of ad
-        int h{};                     ///< Height of ad
+        uint16_t w{};                     ///< Width of ad
```

Comment:
```
#include <cstdint>
```
don't worry if you forget , it won't compile on windows since vc14 does not pre-include it and I will add it later.